src/include/distributed/hash_helpers.h

```diff
@@ -11,6 +11,8 @@
 
 #include "utils/hsearch.h"
 
+#if PG_VERSION_NUM < 120000
+
```

Comment:
Why don't we need this function > 120000?

Reply:
Postgres now exports it. Will add comment